### DISCUSSION TITLE
What Eastern European nation passed a January 2023 law allowing piracy of software whose copyright holders are from "unfriendly nations?"

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#360] [Galaxy Trivia] [answered]
**Hint:** president Alexander Lukashenko

**Comment by peterbaccaro** [comment]
Belarus passed this law on 3rd January 2023.  Alexander Lukashenko, President of Belarus, signed the proposal into law on 3rd January 2023, which means it is now legal in Belarus to access pirated audio-visual material and computer software if the rights holders to the artefact are from “foreign states that commit unfriendly actions against Belarusian legal entities and (or) individuals.” In addition, the law also covers international companies from “unfriendly” nations who hold rights.

**Comment by thuongtruong1009** [comment, marked as answer]
The government of Belarus has passed a law that permits the use of media and intellectual property, such as computer software, in the country without the consent of copyright holders from “unfriendly” foreign nations.
